admin_guide/managing_networking.adoc

```diff
@@ -903,3 +903,50 @@ in all other projects. The project to be isolated must first be configured to
 opt in to isolation by setting the proper annotation on its Namespace object,
 and then creating `NetworkPolicy` objects indicating the incoming connections to
 be allowed.
+
+. Create the `NetworkPolicy` objects:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: extensions/v1beta1
+metadata:
+  name: allow-http-and-https
+spec:
+podSelector:
+ingress:
+- ports:
+  - protocol: TCP
+    - port: 80
+    - port: 443
+----
++
+Optionally, set the `name` parameter to `default-deny` to ensure that the pods that aren't matched to a `NetworkPolicy` continue to be fully-isolated. Delete any existing `NetworkPolicy` objects that previously had the different value. For example, if you are changing the value from `allow-http-and-https` to `default-deny`, delete the old `NetworkPolicy` objects.
+
+. Configure the pod network in the master configuration file at *_/etc/origin/master/master-config.yaml_*:
++
+[source,yaml]
+----
+networkConfig:
+  clusterNetworkCIDR: 10.128.0.0/14 <1>
+  hostSubnetLength: 9 <2>
+  networkPluginName: "redhat/openshift-ovs-networkpolicy" <3>
+  serviceNetworkCIDR: 172.30.0.0/16 <4>
```

Comment:
maybe don't show the `clusterNetworkCIDR` / `hostSubnetLength` / `serviceNetworkCIDR` part and just put `...` instead? To make it clear that all we're really talking about is the `networkPluginName` line. (Likewise below)